### PR TITLE
Added last_updated to check_status route

### PR DESCRIPTION
**Pull request by jworcestBAH**
Check status route now includes last_updated

### CODE REVIEW

File: tests/fileTests.py [hunk 1]
```diff
@@ -18,6 +18,7 @@ class FileTests(BaseTestAPI):
     updateSubmissionId = None
     filesSubmitted = False
     submitFilesResponse = None
+    status_check_time = {"year":2016,"month":01,"day":02,"hour":03,"minute":04,"second":05}
```
Comment (bsweger) [inline tests/fileTests.py]: Is this line being used in the tests anywhere (couldn't find it, but looked pretty quickly)?

**Comment by bsweger** [comment]
@jworcestBAH done reviewing--one small question inline above...

File: tests/fileTests.py (hunk 1 above)
Comment (jworcestBAH) [inline tests/fileTests.py]: Good catch, this was part of my attempt to manually set the last updated time, which didn't work out.  I've removed it now.

**Comment by bsweger** [comment]
Looks good--merging!

**Pull request merged**